eng/Packages.Data.props

```diff
@@ -152,13 +152,13 @@
     <!-- Other approved packages -->
     <PackageReference Update="Microsoft.Azure.Amqp" Version="2.6.7" />
     <PackageReference Update="Microsoft.Azure.WebPubSub.Common" Version="1.2.0" />
-    <PackageReference Update="Microsoft.Identity.Client" Version="4.61.3" />
-    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="4.61.3" />
+    <PackageReference Update="Microsoft.Identity.Client" Version="4.65.0" />
+    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="4.65.0" />
     <!--
       TODO: This package needs to be released as GA and arch-board approved before taking a dependency in any stable SDK library.
```

Comment:
Q: Can this TODO comment be removed now?

Reply:
probably 😄